hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -683,15 +683,29 @@ public void closeContainer(long containerID) throws IOException {
   public Pipeline createReplicationPipeline(HddsProtos.ReplicationType type,
       HddsProtos.ReplicationFactor factor, HddsProtos.NodePool nodePool)
       throws IOException {
+    Map<String, String> auditMap = Maps.newHashMap();
```

Comment:
I see we have used the final keyword for the map variable except here but I would suggest we can remove the final keyword for the map variable in other places as it won't have any effect, if we really want the map values shouldn’t change once initialized we should use `Collections.unmodifiableMap(map)` but I feel we don't need it here, so I would suggest we can remove the final keyword for the map variables in this PR.

Reply:
Thank you for the review and suggestion! I have removed the final keyword form auditMaps now.